docs/resources/Channel.md

```diff
@@ -857,24 +855,20 @@ Any message IDs given that do not exist or are invalid will count towards the mi
 ###### JSON Params
 
 | Field    | Type                | Description                               |
-| -------- | ------------------- | ----------------------------------------- |
+|----------|---------------------|-------------------------------------------|
 | messages | array of snowflakes | an array of message ids to delete (2-100) |
 
-## Bulk Delete Messages (deprecated) % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/bulk_delete
-
-Same as above, but this endpoint is deprecated.
-
 ## Edit Channel Permissions % PUT /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/permissions/{overwrite.id#DOCS_RESOURCES_CHANNEL/overwrite-object}
 
 Edit the channel permission overwrites for a user or role in a channel. Only usable for guild channels. Requires the `MANAGE_ROLES` permission. Returns a 204 empty response on success. For more information about permissions, see [permissions](#DOCS_TOPICS_PERMISSIONS/permissions).
 
 ###### JSON Params
 
-| Field | Type              | Description                                     |
-| ----- | ----------------- | ----------------------------------------------- |
-| allow | integer or string | the bitwise value of all allowed permissions    |
-| deny  | integer or string | the bitwise value of all disallowed permissions |
-| type  | string            | "member" for a user or "role" for a role        |
+| Field | Type   | Description                                     |
+|-------|--------|-------------------------------------------------|
+| allow | string | the bitwise value of all allowed permissions    |
+| deny  | string | the bitwise value of all disallowed permissions |
+| type  | string | "member" for a user or "role" for a role        |
```

Comment:
Why is this still a string, when the API returns numbers? Is that intended, or is that just an oversight? 👀

Reply:
docs oversight, should be number

Reply:
Fixed in latest